### PR TITLE
Fix typo in index.md

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1034](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tutorials/index?branch=pr-en-us-1034)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C19%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609191357336206-1034%5Cworkflow_report.html).

File: docs/csharp/tutorials/index.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ demonstrates web communications, JSON serialization, and Object Oriented
 features in the C# language.
 * [Working with LINQ](working-with-linq.md) This tutorial demonstrates many of the features of LINQ and the language elements that support it.
 
-* [Microservices hosted in Docker](microservices.md) This tutorial demonstrates building an asp.net core microservice and hosting it it Docker.
+* [Microservices hosted in Docker](microservices.md) This tutorial demonstrates building an asp.net core microservice and hosting it in Docker.
```
Comment (badersur) [inline docs/csharp/tutorials/index.md]: I wasn't aware of this and I don't know how to update the PR! Should I issue another PR and close the previous one?!

**Comment by BillWagner** [comment]
@badersur Thank you for contributing. We really appreciate it.

This looks great, and I'll :shipit: as soon as the CI build finishes.

**Comment by badersur** [comment]
@BillWagner You are welcome. I'm planning to read .NET Core and C# Guide so I may issue more PRs soon :)

**Pull request merged**